athena-redis/src/main/java/com/amazonaws/athena/connectors/redis/integ/RedisIntegTestHandler.java

```diff
@@ -66,8 +68,8 @@ public final void handleRequest(InputStream inputStream, OutputStream outputStre
         RedisConnectionWrapper<String, String> standaloneConnection = null;
         RedisConnectionWrapper<String, String> clusterConnection = null;
         try {
-            standaloneConnection = connectionFactory.getOrCreateConn(standaloneConnectionString, false, false);
-            clusterConnection = connectionFactory.getOrCreateConn(clusterConnectionString, true, true);
+            standaloneConnection = connectionFactory.getOrCreateConn(standaloneConnectionString, false, false, STANDALONE_REDIS_DB_NUMBER);
+            clusterConnection = connectionFactory.getOrCreateConn(clusterConnectionString, true, true, null);
```

Comment:
nit: Related to above comment -- I think `null` here is a little weird. It's really "default 0" rather than "nonexistent".

Reply:
Well this is just for integ test. Could really be anything since cluster doesn't do anything with the dbnumber

Reply:
fixed